src/example-types/date.js

```diff
@@ -90,7 +81,7 @@ let hasValue = ({ from, to, range }) =>
   range !== 'allDates' &&
   ((range === 'exact' && (from || to)) || range !== 'exact')
 
-module.exports = {
+export default {
```

Comment:
`module.exports = { foo }` is like `export let foo = ...`, not like `export default { foo }

Reply:
I get a warning when trying to run the esbuild scripts with "module" set.  I don't know if I should convert back or not